.github/renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    ":disableDependencyDashboard",
+    ":semanticCommitsDisabled"
+  ],
+  "prHourlyLimit": 10,
+  "gomod": {}, // Upgrade go dependencies.
+  "github-actions": {}, // Upgrade GitHub actions.
+  // Renovate evaluates all packageRules and does not stop after the first match.
+  // Rules that appear later in this list override earlier rules.
+  "packageRules": [
+    // Group all Go dependencies in the same PR.
+    {
+      "matchManagers": ["gomod"],
+      "groupName": "go dependencies"
+    },
+    // Separate all kubernetes dependencies to a different PR.
+    {
+      "matchManagers": ["gomod"],
+      "matchPackagePrefixes": ["k8s.io/"],
+      "groupName": "go k8s libraries"
+    },
+    // Separate k6 dependencies to a different PR.
+    {
+      "matchManagers": ["gomod"],
+      "matchPackagePrefixes": ["go.k6.io/"],
+      "groupName": "k6 core"
+    },
```

Comment:
Goja is a challenge here as it is not versioned. I haven't figured out a way to tell renovate to use whatever is used upstream.

This could be done by hand though, by creating an ad-hoc cron job that fetches the `go.mod` file for the k6 repo and updates goja to match.

Reply:
@pablochacin What are your thoughts on this? Right now, goja is bundled with the rest of go dependencies. Should we give it a special treatment, either by putting it on a separate PR or for working a bit more on the sync approach?

Reply:
I think we should address goja in a follow-up PR and give us some time to consider alternatives.

Reply:
Makes sense, I'll leave it on the `go dependencies` group for now then.